openspec/changes/codebase-import-runtime-hardening/specs/bundle-packaged-resources/spec.md

```
@@ -0,0 +1,17 @@
+## MODIFIED Requirements
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)